avionics/envs/x86/include/sensors/barometer.h

```diff
@@ -8,6 +8,7 @@
 #include "../env_config.h"
 #include "../sensor_data.hpp"
 #include "sensors_interface.h"
+#include <bits/stdint-uintn.h>
```

Comment:
bits is a GCC specific header - should instead use [`cstdint`](https://en.cppreference.com/w/cpp/header/cstdint)